lops/lop-domain-a72.dts

```diff
@@ -26,12 +26,12 @@
                 lop_2 {
                         // modify to "nothing", is a remove operation
                         compatible = "system-device-tree-v1,lop,modify";
-                        modify = "/cpus_microblaze1::";
+                        modify = "/cpus-cluster@1::";
                 };
                 lop_3 {
                         // modify to "nothing", is a remove operation
                         compatible = "system-device-tree-v1,lop,modify";
-                        modify = "/cpus_microblaze2::";
+                        modify = "/cpus-cluster@2::";
```

Comment:
System device-tree now generating the microblaze cpu node names as cpus_microblaze1 and cpus_microblaze2, No need of this change

Reply:
see above comment